c10/core/UndefinedTensorImpl.cpp

```diff
@@ -24,6 +24,10 @@ int64_t UndefinedTensorImpl::dim() const {
   AT_ERROR("dim() called on undefined Tensor");
 }
 
+bool UndefinedTensorImpl::has_storage() const {
+  return false;
```

Comment:
all the other functions here just throw an exception, why is this one different?

Reply:
Fixed